src/elements/Text.tsx

```diff
@@ -42,10 +43,10 @@ Text.preserveSoftBreaks = function (text: string): ReactNode {
                 ? [part]
                 : [
                       ...result,
-                      <>
+                      <Fragment key={part}>
```

Comment:
> `key={part}` 

This looks like a very bad idea. There's no guarantee that `part` will be unique. Providing a duplicate key is way worse than not providing one. 

Should we use the array index instead?

Reply:
a642c5e0d2e05601bda2805e88aad519a1446fee